src/utils/common.ts

```diff
@@ -244,22 +244,27 @@ export function retry<T>(call: () => Promise<T>, times: number, delayS: number):
   return promiseChain;
 }
 
+export type TransactionCostEstimate = {
+  nativeGasCost: BigNumber; // gas
+  tokenGasCost: BigNumber; // wei/gas
```

Comment:
How can this be in Wei or gas? Or am I misunderstanding the slash? Can you make this comment a bit more clear

Reply:
Bad comment - good catch. Fixed: https://github.com/across-protocol/sdk-v2/pull/453/commits/e7398df1162e0243b065dfe98d2e1a297fee88ca